kong/plugins/httplog/schema.lua

```diff
@@ -0,0 +1,6 @@
+return {
+  logging_url = { required = true, type = "string" },
+  method = { require = false, default = "POST", enum = { "POST", "PUT", "PATCH" } },
```

Comment:
I think the `enum` looks fine and will avoid that 99.9% of the users could write a typo instead (if it's a free form field, we can't prevent typos on the most common methods). For the remaining 0.01% who has a need to use their own HTTP method then we are open to change this.

`GET`, `TRACE`, `CONNECT`, `HEAD` and `OPTIONS` are excluded on purpose because whoever wants to use these methods for logging should be discouraged from doing so.